### PR TITLE
Features/tcccd 63

### PR DESCRIPTION
**Pull request by sadierichardson**

### CODE REVIEW

File: R/acf_dm.R [hunk 1]
```diff
@@ -85,14 +86,18 @@ assign_acf_class <- function(pth,
...
+                                 family_fips_code = "FIPS",
+                                 date = "ReportingDate"), 
                "Children"  = list(child_id = "ChildrenID",
-                                  child_age = "Age"), 
+                                  child_age = "Age",
+                                  date = "ReportingDate"), 
                "Providers"  = list(operation_number = "Data.StateID",
-                                   provider_zip = "Data.ZipCode"))), 
+                                   provider_zip = "Data.ZipCode",
+                                   date = "Data.ReportingDate"))),
```
Comment (holmesjoli) [inline R/acf_dm.R:100]: @sadierichardson why do we need to add in the Data.ReportingDate?

Comment (sadierichardson) [reply R/acf_dm.R:100]: @holmesjoli There are three different reporting dates in each quarter, so I think we need to match family to provider using reporting date as well. I also am unsure if the child ids are unique across or within reporting dates so I added it in to investigate. But we're not using the ACS data for much anymore so I don't think it matters.

Comment (holmesjoli) [reply R/acf_dm.R:100]: Okay! Just wanted to double-check!

File: R/frontline_dm.R [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+#' @param format string. The format to use to parse the string. Default is
+#' '%m.%d.%y'.
+#' @return date
+parse_date.frontline <- function(x, format = "\\d{4}-\\d{2}-\\d{2}"){
+
+  date <- lubridate::ymd(stringr::str_extract(x, format))
+}
+
+#' @title Data management for date column
+#' @description Manage date
```
Comment (holmesjoli) [inline R/frontline_dm.R:11]: @sadierichardson let's add some detail about why we filter the date to be less than the export_date in the description

Comment (sadierichardson) [reply R/frontline_dm.R]: @holmesjoli done! let me know if that makes sense.

**Comment by holmesjoli** [comment]
@sadierichardson I'm getting an error "earlyed_pre_enrollment" doesn't exist when I try and run the function to subset the data

**Comment by sadierichardson** [comment]
> @sadierichardson I'm getting an error "earlyed_pre_enrollment" doesn't exist when I try and run the function to subset the data

@holmesjoli I changed the pre-k file in the 63-supply-fix ticket, so I bet that's why. Maybe look at that ticket first or re-run the child_care_db from this ticket so that they match. If that's not the issue, let me know and I can look into it more!

**Review by holmesjoli — APPROVED** [review]

**Pull request merged**